src/Helpers/MemoryHelper_Room.ts

```diff
@@ -535,4 +538,45 @@ export default class MemoryHelper_Room {
         // * Optionally apply a filter or otherwise check the limits before assigning them
         Memory.rooms[room.name].creepLimit!.remoteLimits = newLimits;
     }
+
+    /**
+     * update the memory which contains the center of the bunker for the room
+     * @param room the room we are in
+     */
+    public static updateBunkerCenter(room: Room): void {
```

Comment:
Not a big fan of how we do this right now. Likely to change

Reply:
Extremely open to change on this one. Had a tough time figuring this out